enzy_htp/_interface/pymol_interface.py

```diff
@@ -278,27 +278,50 @@ def export_pdb(self, pymol_session: pymol2.PyMOL, pymol_obj_name: str, if_retain
         if if_retain_order:
             pymol_session.cmd.set("retain_order")
         pymol_session.cmd.save(pymol_outfile_path, pymol_obj_name)
-
+        
         return pymol_outfile_path
 
-    def export_enzy_htp_stru(self, 
-                             pymol_session: pymol2.PyMOL,
-                             pymol_obj_name: str,        
-                             if_retain_order: bool = False) -> Structure:
+
+    def export_enzy_htp_stru(self, pymol_obj_name: str,
+                            pymol_session: pymol2.PyMOL,
+                            if_retain_order: bool = False,
+                            if_fix_naming: bool = False) -> Structure:
         """
         Saves a PyMOL object to a Structure object.
         Args:
             pymol_session: the target PyMOL session.
             pymol_obj_name: the name of the target enzyme in PyMOL.
             if_retain_order: if the saving should keep the order of the atoms in the original object.
+            if_fix_naming: if the PyMOL naming of atoms should be updated to match the Structure convention.
```

Comment:
Also state the problem for both of them. Giving a link pointing to the two issue each is a good idea